apps/cardiovascular-health/src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<fhir-chart-layout [toolbar]="['loading', 'options']">
+<fhir-chart-layout [toolbar]="['loading', 'options']" [showLayerOptionPanel]="false">
```

Comment:
The name of this input is misleading because it sounds like it will hide the whole options panel, but it does not. Maybe we should call it "showAdvancedOptions" instead?

Reply:
Okay @semicolin I will update that.